lib/features/steps/runner_steps.rb

```diff
@@ -62,3 +62,65 @@
   $logger.warn 'Sleep was used! Please avoid using sleep in tests!'
   sleep(seconds)
 end
+
+# Starts an interactive terminal
+When('I start a new terminal') do
+  Runner.stop_interactive_session
+  Runner.get_interactive_session
```

Comment:
I think this can wait for it to be ready for input by sleeping until it has a `pid`, e.g.

```suggestion
  cli = Runner.get_interactive_session
  attempts = 0

  until cli.pid
    raise "did not start in time" if attempts > 10 # or some number that makes more sense

    sleep(0.1) # or whatever
    attempts += 1
  end
```

Reply:
Yes, this was something I was planning to do (to remove all the waits).